App/Sagas/TextileSagas.ts

```diff
@@ -150,11 +150,17 @@ export function * backgroundLocationPermissionsTrigger() {
   }
 }
 
-export function * addPhotoLike(action: ActionType<typeof UIActions.addLikeRequest>) {
+export function * addPhotoLike(action: ActionType<typeof UIActions.addLike.request>) {
   const { blockId } = action.payload
   try {
-    yield call(Textile.likes.add, blockId)
+    const likingPhotos = yield select(UISelectors.likingPhotos)
+    // If photos is already being liked, don't let the user like it again
+    if (Object.keys(likingPhotos).indexOf(blockId) !== -1) {
```

Comment:
Maybe I'm reading this wrong, but....
1. You get the list of block ids in the process of being liked
2. You check if that list already contains the block id trying to be liked now
3. If the new block id isn't in the list, `indexOf` should return -1
4. But you're checking that it returns something other than -1

Reply:
Okay, this is totally wrong, good catch but it begs the question-How was it working for me locally?? I must have already populated the state before setting this or something.

Reply:
🤷‍♂